autocertkit/utils.py

```diff
@@ -65,6 +65,7 @@
 
 # Capability Tags
 REQ_CAP = "REQ"
+REQ_CAP_MULTICAST = "REQ_MULTICAST"
```

Comment:
This is not needed unless you want to add logic to handle this capability in report.